src/MSALWrapper/TokenResult.cs

```diff
@@ -32,6 +32,22 @@ public enum AuthType
         IntegratedWindowsAuthenticationFlow,
     }
 
+    /// <summary>

```

Comment:
My other main comment is about this - we should not need this, and instead can better encode the concept of interaction by using better types.